tests/misc_env/install_prereq.py

```diff
@@ -139,7 +139,8 @@ def registry_login(ceph, distro_ver):
         docker for RHEL 7.x and podman for RHEL 8.x'''
     cdn_cred = get_cephci_config().get('cdn_credentials')
     if not cdn_cred:
-        log.warn('cdn_credentials not found')
+        log.warn('no cdn_credentials in ~/.cephci.yaml.'
```

Comment:
since `warn` is deprecated in py3, why not use `warning`

Reply:
Sure. I will submit that in a separate PR after we merge this, since that's a separate logical change.